qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/EditDemographicsAndContactInfoSteps.java

```diff
@@ -20,95 +24,132 @@
 
 import org.openmrs.contrib.qaframework.RunTest;
 import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.ActiveVisitsPage;
 import org.openmrs.contrib.qaframework.page.FindPatientPage;
 import org.openmrs.contrib.qaframework.page.RegistrationEditSectionPage;
-import org.openmrs.contrib.qaframework.page.RegistrationSummaryPage;
 
 public class EditDemographicsAndContactInfoSteps extends Steps {
 
+	private String prefix;
 	private TestData.PatientInfo testPatient;
+	private ActiveVisitsPage activeVisitsPage;
 	private RegistrationEditSectionPage registrationEditSectionPage;
-	private RegistrationSummaryPage registrationSummaryPage;
+	private FindPatientPage findPatientPage;
+	private static final String VISIT_TYPE_UUID = "7b0f5697-27e3-40c4-8bae-f4049abfb4ed";
```

Comment:
Is this UUID  auto generated, How are you generating this uuid.

Reply:
i have also struggled to understand how tht figure came about here --> https://github.com/openmrs/openmrs-contrib-qaframework/blob/56c2c15f535d8a1e897cdcc0d2928b30620181f1/qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/ContactInfoTest.java#L26

Reply:
Thats fine . that uuid is simply a string concatenated via testData class.

Reply:
how about `new TestData.TestVisit(testPatient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();

Reply:
Seems you are getting visitTypeUuid. isn't it part of the TestData attribute. Doe it work that way.

Reply:
the test passes but anyway lemi keep the hardwired one for now